include/RevRegFile.h

```diff
@@ -255,8 +268,10 @@ class RevRegFile {
   void AdvancePC(T bytes) {
     if ( IsRV32 ) {
       RV32_PC += bytes;
+      TRACE_PC_WRITE(RV32_PC);
     }else{
       RV64_PC += bytes;
+      TRACE_PC_WRITE(RV64_PC);
```

Comment:
If you want to avoid unnecessary tracer output advancing `PC` at the end of every instruction, we can refactor this so that `AdvancePC` only advances one instruction and avoids excessive tracer output, and use `SetPC` for all other `PC` modifications.